lib/user.rb

```diff
@@ -0,0 +1,22 @@
+class User
+# attr_reader :name
+attr_accessor :jokes, :name
```

Comment:
This challenge only required the use of an `attr_reader`. We don't want to expose our internal state to the outside world unless we absolutely need to.